samples/word/35-ranges/get-pages.yaml

```
@@ -0,0 +1,244 @@
+order: 4
+id: word-ranges-get-pages
+name: 'Work with pages, panes, and windows'
+description: 'Shows how to work with pages, panes, and windows APIs.'
```

I think that we're working with APIs is already implied since we're in Script Lab. 
```suggestion
description: 'Shows how to work with pages, panes, and windows.'
```